advanced/sr_gui_movement_recorder/src/sr_gui_movement_recorder/movement_recorder.py

```diff
@@ -370,7 +373,8 @@ def __init__(self, context):
         # selecting the first available hand
         self.__selected_hand = None
         # TODO(@dg-shadow) Fix hand finder etc to work with hand H
-        self.hand_selected(self.hand_parameters.mapping.keys()[0])
+        if hand_finder.hand_e_available():
+            self.hand_selected(self.hand_parameters.mapping.keys()[0])
```

Comment:
add 
```python
else:
    self.hand_commander=SrHandCommander()
```

otherwise the hand_commander won't get instantiated... You should have tested this the final version before doing a PR - you would have noticed this if you had done ;)